irc3/plugins/command.py

```diff
@@ -268,7 +268,7 @@ def split_command(self, data, use_shlex=True):
             if use_shlex:
                 return shlex.split(data)
             else:
-                return data.split()
+                return data.split(' ')
```

Comment:
You can use @command(use_shlex=False) but yes, you're right, it's ok since using shlex is the default